### ISSUE TITLE
deploy.bat Deleted all steam games

### ISSUE THREAD

**Issue by Kevino1** [Jacob-Mango/DayZ-Community-Framework#3]
deploy.bat just deleted all of my steam games, and probably some other things aswell

**Comment by Kevino1** [comment]
> deploy.bat just deleted all of my steam games, and probably some other things aswell

just realised it deleted almost everything on my F: Drive which is where my most of my stuff was

**Comment by Kevino1** [comment]
> > deploy.bat just deleted all of my steam games, and probably some other things aswell
> 
> just realised it deleted almost everything on my F: Drive which is where my most of my stuff was

![image](https://user-images.githubusercontent.com/46437266/50753183-1273e600-12a5-11e9-8853-cad2f6f7b2a5.png)
there used to be probably 150 files, images, etc in my F: drive, and this is what there is left

**Comment by Jacob-Mango** [comment]
I'm sorry for what happened but I am not responsible here for your loss of files.

The best you can do is to not touch the drive any further and send it to a data recovery specialist or if you know how to, recover the data yourself. There are some programs online that could help with that. 

It is unfortunate what happened but the deploy.bat has launch params that must be used which you must not have set correctly. I can not recover the files for you.

Jacob

**Comment by LordRampantHump** [comment]
while you are not directly responsible you should take that sh1t bat off

**Comment by Liven28** [comment]
Same thing!
You have really push a bat file which delete user files without prompting, without warning! 
Not even a readme file warning.
And you are "not responsible" for that?
You're kidding me?!

**Comment by Jacob-Mango** [comment]
As of last month I have updated the batch files so they won't do that any more.

**Comment by Liven28** [comment]
Sorry but I had this problem yesterday (with files cloned yesterday)

It deleted all what I had in directory that contain the unzip directory. For exemple:

C:\MypersonnalData\MyDownloasd\DayZ-Community-Framework-master\Examples

All the files and directorys in "MyDownloads" have been deleted (the .bat being in exemple directory). 
I losted beetween 100 and 200 Go of data, luckily nothing realy importante (nothing I can remember yet), but I can't imagine if I had decided to unzip on a hard drive root.

You realy should look again on that file or at least make a "warning" txt file beside this bat file. I launched it after a (too) quick look in the .bat thinking it will just generated a .pbo from the exemple.... And surprise!

Hope you'll find a safe solution becouse I realy love the work you make (I'm a daily user of CoT).
